packages/flutter/lib/src/material/refresh_indicator.dart

```diff
@@ -308,7 +315,7 @@ class RefreshIndicatorState extends State<RefreshIndicator> with TickerProviderS
         indicatorAtTopNow = true;
```

Comment:
Not totally following the logic here, but if the `AxisDirection.up` sets `indicatorAtTopNow` to `widget.reverse`, shouldn't this be setting it to `!widget.reverse`?

Reply:
I agree, it may be easier to follow if the widget state is updated to reflect that the indicator is leading or trailing. A getter may be more readable.

Reply:
Thanks so much for feedback! I will update this soon